src/ingestor.ts

```diff
@@ -79,6 +80,16 @@ export default async function ingestor(req) {
     if (!Object.keys(hashTypes).includes(hash)) return Promise.reject('wrong types');
     type = hashTypes[hash];
 
+    try {
+      if (
+        (message.space.startsWith('s:') || !message.space.includes(':')) &&
```

Comment:
Not sure how we missed before but it should throw an error if `message.space` is undefined no?

Reply:
I believe the schema validation is already ensuring `message.space` is always set

Reply:
No 🙈 it is optional. but something else is preventing it not to throw an error 🤔

Reply:
I will run few tests and see. but else it looks good, we can merge

Reply:
I tried to edit profile with this change (alias request) and it fail 
```ts
undefined
TypeError: Cannot read properties of undefined (reading 'startsWith')
```

Reply:
how it was working before 😅 with https://github.com/snapshot-labs/snapshot-sequencer/pull/417/files

Reply:
Maybe we could skip the check if `!message.space`